samples/snippets/quickstart.py

```diff
@@ -73,9 +73,15 @@ def list_all_instances(
         iterable collections of Instance objects as values.
     """
     instance_client = compute_v1.InstancesClient()
-    agg_list = instance_client.aggregated_list(project=project_id)
+    # Requesting a list of Instances, aggregated by the zone they are located in.
+    # The max_results parameter tells the API to return no more than 5 results per response page.
```

Comment:
Recommend making the comment less number-specific. Maybe something like this:

```suggestion
    # Use the max_results parameter to limit the number of results that the API returns per response page.
```